src/ontology/edits/oeo-model.omn

```diff
@@ -609,11 +609,16 @@ Class: OEO_00000276
 Class: OEO_00000277
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A model factsheet is a factsheet that contains information about a model."^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A model factsheet is a model descriptor that contains a brief description of all relevant model information."^^xsd:string,
```

Comment:
You changed the definition for the "factsheet"-`model factsheet`. Is that on purpose?

Reply:
Yes, its the def from issue #82

Reply:
> And could you please remove the German language labels?

Done. Thanks.